src/EditorFeatures/CSharpTest/Completion/CompletionProviders/TypeImportCompletionProviderTests.cs

```diff
@@ -1413,6 +1415,138 @@ private static void AssertRelativeOrder(List<string> expectedTypesInRelativeOrde
             }
         }
 
+        [InlineData(true)]
+        [InlineData(false)]
+        [Theory, Trait(Traits.Feature, Traits.Features.Completion)]
+        public async Task TestBrowsableAwaysFromReferences(bool isProjectReference)
+        {
+            var srcDoc = @"
+class Program
+{
+    void M()
+    {
+        $$
+    }
+}";
+
+            var refDoc = @"
+namespace Foo
```

Comment:
References of "Foo" was removed from the code base a very long time ago in https://github.com/dotnet/roslyn/pull/21224

I'm not sure if you still don't want to include "Foo"s or not. But just noting.

Reply:
@CyrusNajmabadi Is this ban on `Foo` still a thing? There's a ton of them in our tests still.

Reply:
it's likely this will flag a review check at some point.  so probably best to just not introduce new instances.